### PR TITLE
Component create and delete

### PR DESCRIPTION
**Pull request by kadel**
#14

### CODE REVIEW

**Comment by kadel** [comment]
creating new component from remote git repository should work now.

`ocdev component create nodejs --git https://github.com/openshift/nodejs-ex.git`

File: pkg/application/application.go [hunk 1]
```diff
@@ -52,3 +55,28 @@ func Create(name string) error {
 
 	return nil
 }
+
+// GetCurrentApplication gets currently active application
+// if no application is active, defaultApplication is created
+func GetCurrentApplication() (string, error) {
```
Comment (containscafeine) [inline pkg/application/application.go]: I think should be a part of the config package in config.go, like we have other methods there like, `ApplicationExists()`, `AddApplication`, we can also have `CurrentApplication()`

Comment (kadel) [reply pkg/application/application.go]: I think that if it is operation on application that it should be in the application package. 
Doing it like this, if we move to another way to store application info we won't have to touch anything else than this one function

**Comment by kadel** [comment]
@containscafeine it should be ready to merge.

following should work
```
ocdev component create nodejs --git https://github.com/openshift/nodejs-ex.git
ocdev component create nodejs --dir ./nodejs-ex
ocdev component create nodejs
```

File: pkg/component/component.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	if err != nil {
+		return "", errors.Wrap(err, "unable to get current application")
+	}
+
+	output, err := occlient.NewAppS2I(name, ctype, url, map[string]string{applicationLabel: currentAppliction, componentLabel: name})
+	if err != nil {
+		return "", err
+	}
+
+	SetCurrent(name)
```
Comment (containscafeine) [inline pkg/component/component.go]: you maybe forgot to store this to `err` here

File: pkg/component/component.go [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+	currentAppliction, err := application.GetCurrent()
+	if err != nil {
+		return "", errors.Wrap(err, "unable to get current application")
+	}
+
+	output, err := occlient.NewAppS2IEmpty(name, ctype, map[string]string{applicationLabel: currentAppliction, componentLabel: name})
+	if err != nil {
+		return "", err
+	}
+	SetCurrent(name)
```
Comment (containscafeine) [inline pkg/component/component.go]: same

**Pull request merged**